documentation/docs/contributing/development-guide/coding/api/api.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Still an issue

Reply:
Not sure if we can do anything about it, since if you want to have a clickable folder with content, you need to set sidebar_position on top of the document.  And for folders that you want to have "generated-index" link type you can use .json or .yaml _category_ file.

Reply:
I want to discuss this further